robot_smach_states/test/human_interaction/human_interaction.py

```diff
@@ -51,3 +52,61 @@ def test_say(self):
         with self.assertRaises(RuntimeError) as cm:
             say.execute()
             self.assertIn("Not all named place holders are provided", cm.exception)
+
+
+class TestHearOptions(unittest.TestCase):
+
+    def setUp(self):
+        self.robot = Mockbot()
+
+    def test_hear_options_success(self):
+        options = ["apple"]
+        hear = HearOptions(self.robot, options)
+        self.assertEqual(hear.execute(), "apple")
+
+    def test_hear_options_fail(self):
+        class HMIMock(object):
+            def query(self, description, target, grammar, timeout):
+                raise TimeoutException
+
+        options = ["apple"]
+        self.robot.hmi = HMIMock()
```

Comment:
Should this be part of the stand MockBot?

Reply:
IMHO it shouldn't: it's a specific implementation to raise a TimeoutException to see what happens